lib/internal/Magento/Framework/Cache/Exception/WrongConfigurationException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Framework\Cache\Exception;
+
+/**
+ * Exception for situation when cache configuration is wrong
+ *
+ * @api
+ */
+class WrongConfigurationException extends \Exception
```

Comment:
Sorry, I can not find place where this kind of exception is used...
Could you advice?